src/components/yemen/YemenPages.jsx

```diff
@@ -18,12 +20,12 @@ class YemenPages extends Component {
     return (
       <div>
         <Switch>
-          <Route path="/yemen/education/crisis">
-            Maya Article
+          <Route path="/yemen/education/juhood">
+            <MayaArticle />
           </Route>
 
           <Route path="/yemen/education/resources">
-            Other Resources List
+            <Resources />
```

Comment:
I think it would be better if you called this component, `YemenResources`. We will be having many pages that are resources for different issues, in the future, so, its best to specify.